docs/agent-reference/how-to-x509-authentication.md

```diff
@@ -97,8 +98,8 @@ For IoT Edge scenarios where the ADU agent runs as a module, configure module id
                 "connectionX509PrivateKeyFilePath": "/etc/adu/certs/module-client.key",
                 "connectionX509CaCertFilePath": "/etc/adu/certs/ca.pem"
             },
-            "manufacturer": "Contoso",
-            "model": "Smart-Box"
+            "manufacturer": "device_info_manufacturer",
+            "model": "device_info_model"
```

Comment:
any reason why `manufacturer` and `model` appear in top level as well as underneath `agents`?